packages/sdk/modules/base/BaseModule.ts

```diff
@@ -1,30 +1,30 @@
 import { type Address, type Hex, encodeFunctionData, parseAbi } from "viem"
 import contracts from "../../__contracts/index.js"
-import type { SmartAccountSigner } from "../../account/index.js"
+import type { Holder } from "../../account/utils/toHolder.js"
+import type { Module } from "../../clients/decorators/erc7579/index.js"
 import {
-  type Module,
   type ModuleType,
   type ModuleVersion,
   moduleTypeIds
 } from "../utils/Types.js"
 
 export abstract class BaseModule {
-  moduleAddress: Address
-  data: Hex
+  address: Address
+  context: Hex
   additionalContext: Hex
   type: ModuleType
   hook?: Address
   version: ModuleVersion = "1.0.0-beta"
   entryPoint: Address = contracts.entryPoint.address
-  signer: SmartAccountSigner
+  holder: Holder
```

Comment:
holder term comes from viem?

Reply:
No it's a term I made up myself. each account seemed to have their own term for this. 'signer' sounded like an ethers term so just wanted to avoid using that.